tests/rpmbuild.at

```diff
@@ -131,6 +131,37 @@ runroot rpmbuild -bs /data/SPECS/specstep.spec 2>&1|grep ^Executing|cut -d: -f1
 [])
 AT_CLEANUP
 
+# ------------------------------
+# Check if rpmbuild -ba *.spec works after 2038
+AT_SETUP([rpmbuild -ba *.spec after 2038])
+AT_KEYWORDS([build])
+RPMDB_INIT
+AT_CHECK([[
+touch -t 204205042355 ${RPMTEST}/data/SPECS/hello.spec
+touch -t 204205042355 ${RPMTEST}/data/SOURCES/hello-1.0.tar.gz
+touch -t 204205042355 ${RPMTEST}/data/SOURCES/hello-1.0-modernize.patch
+
+unset SOURCE_DATE_EPOCH
+run rpmbuild --quiet \
```

Comment:
Put the rpmbuild run after the RPMDB_INIT and before the first AT_CHECK() to make the result available to all sub-tests (see below)